compliance_checker/suite.py

```diff
@@ -358,13 +367,16 @@ def standard_output(self, ds, limit, check_name, groups):
 
         # Let's add the version number to the check name if it's missing
         check_name = self._get_check_versioned_name(check_name)
+        check_url = self._get_check_url(check_name)
+        _len_ = 120
```

Comment:
Could this be set to `2 * self.col_width` instead?
And could all the other hard-coded numbers below be replaced with a reference to this variable?
(Then the column width could even be an optional command-line argument, but that's for another PR...)

Reply:
@mhidas I made the change to line 371, but I'm quite certain the string alignment in the following lines has to be specified in each string as dictated in 

https://docs.python.org/3.4/library/string.html#formatstrings

Reply:
The documentation doesn't elaborate on this possibility, but there are examples that show you can specify any of the formatting fields from variables (see last section under https://docs.python.org/3.4/library/string.html#format-examples). So you could do e.g.
```python
        print('{:^{width}}'.format("IOOS Compliance Checker Report", width=_len_))
        print('{:^{width}}'.format(check_name + ' check', width=_len_))
        print('{:^{width}}'.format(check_url, width=_len_))
```

Reply:
:man_facepalming: How did I miss those... I'll get those changes in ASAP.